File: frontend/src/members/get-member-response.ts
module members {
  export class GetMemberResponse {
    constructor(data: any) {
      this.member = new Member(data.member);
      this.messages = data.messages.map(m => new MemberMessage(m));
    }

    member: Member;
    messages: Array<MemberMessage>;
  }
}

File: api/src/MyAthleticsClub.Core/Members/GetMember/GetMemberResponse.cs
using System.Collections.Generic;

namespace MyAthleticsClub.Core.Members.GetMember
{
    public class GetMemberResponse
    {
        public GetMemberResponse(Member member, IEnumerable<MemberMessage> messages)
        {
            Member = member;
            Messages = messages;
        }

        public Member Member { get; }

        public IEnumerable<MemberMessage> Messages { get; }
    }
}
